pkg/framework/aries/defaultopt/defaultopt.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package defaultopt
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries"
+	"github.com/hyperledger/aries-framework-go/pkg/storage/leveldb"
+)
+
+// WithStorePath return new default store provider instantiate with db path
+func WithStorePath(storePath string) aries.Option {
+	return func(opts *aries.Aries) error {
+		storeProv, err := leveldb.NewProvider(storePath)
+		if err != nil {
+			return fmt.Errorf("leveldb provider initialization failed : %w", err)
+		}
+		aries.WithStoreProvider(storeProv)
```

Comment:
shouldn't we return the result of this call instead of nil?

Reply:
done